sdks/java/container/Dockerfile

```diff
@@ -48,4 +48,7 @@ RUN if [ "${pull_licenses}" = "false" ] ; then \
     rm -rf /opt/apache/beam/third_party_licenses ; \
    fi
 
+# Add Google Cloud Profiler agent.
+COPY target/profiler/* /opt/apache/beam/
```

Comment:
Let's put this in a directory that is clearly dataflow-specific, like `google_cloud_dataflow` or `google_cloud_profiler`, etc.

Reply:
Done.